include/spotify/json/decode_exception.hpp

```diff
@@ -31,8 +31,9 @@ namespace json {
 class decode_exception final : public std::runtime_error {
  public:
   template <typename string_type>
-  json_never_inline decode_exception(const string_type &what, const size_t offset)
-      : runtime_error(what), _offset(offset) {}
+  json_never_inline explicit decode_exception(const string_type &what, const size_t offset = 0)
```

Comment:
It seems like this default argument was just added for a single test case. I'd prefer not having it here, but instead just set offset to 0 in the test case.